be/src/formats/parquet/level_builder.cpp

```diff
@@ -108,7 +108,9 @@ void LevelBuilder::_write_column_chunk(const LevelBuilderContext& ctx, const Typ
         break;
     }
     case TYPE_CHAR:
-    case TYPE_VARCHAR: {
+    case TYPE_VARCHAR:
+    case TYPE_BINARY:
```

Comment:
better to write a new function to handle binary and varbinary

Reply:
Fixed